### ISSUE TITLE
Implement 'change_count' for html_diff_render and links_diff

### ISSUE THREAD

**Issue by danielballan** [edgi-govdata-archiving/web-monitoring-processing#137]
In both cases, it's not immediately obvious to me how to extract the _number_ of changes efficiently, so I did not address it as part of #136. It's easy enough to address them in a separate PR, but I'm also perfectly happy to sort it out before #136 is merged if people prefer that.

**Comment by Mr0grog** [comment]
I don’t have strong feelings on doing it separately or together. I think for both of them (since they use the same underlying algorithm), the right answer is to count the number of `ins`/`del` elements.

Once I start to get #129 feeling good, I’ll see about having its underlying algorithm return a count. I think this is just a matter of [counting opcodes](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/51d0209bd5f29a3ad2c8db47cd23ce3371be9318/web_monitoring/html_diff_render.py#L294).

Since we are also returning multiple diffs in that one, it might make sense to return counts like:

```py
{
  'combined': 'actual diff...',
  'insertions': 'actual diff...',
  'deletions': 'actual diff...',
  'change_count': 5,
  'insertions_count': 3,
  'deletions_count': 2
}
```

**Comment by Mr0grog** [comment]
Whoops, this was fixed as part of #129. Should have closed it then!
